src/Raven.Client/Documents/Commands/Batches/RevisionsCommandData.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Session;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Client.Documents.Commands.Batches
+{
+    public class RevisionsCommandData : ICommandData
```

Comment:
Can we change the name? Not sure what would be suitable... `CreateRevisionCommandData` ?

Reply:
imho - change to - **ForceRevisionCommandData**
because this command is created only when coming from the 'Force Revision' flow
What do you think?

Reply:
fine